src/components/pages/Work.jsx

```diff
@@ -16,22 +16,20 @@ import classNames from './Work.scss'
 export const WorkById = ({ id }) => {
   const { useApi, poetNodeApi } = useContext(ApiContext)
   const work = useApi('workGetById', id)
-  const [claims, setClaims] = useState(false)
+  const [claims, setClaims] = useState([])
 
   useEffect(() => {
     if (poetNodeApi) {
-      poetNodeApi.graph.get(id).then(setClaims)
+      poetNodeApi.graph.get(encodeURIComponent(`poet:claims/${id}`)).then(setClaims)
```

Comment:
Maybe just `encodeURI`?

Reply:
Also, still need to make this backwards compatible. API will return `[]` for all claims created so far. 

We'll need to check `work.archiveUrl`. If it's present, add a fake graph edge on the fly, where `archiveUrl` is the root of the DAG and `work` points to it.